src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.ts

```diff
@@ -815,47 +820,48 @@ export class QuizParticipationComponent implements OnInit, OnDestroy {
+            const jhiAlert = this.jhiAlertService.error(errorMessage);
+            jhiAlert.msg = errorMessage;
             this.unsavedChanges = true;
             this.isSubmitting = false;
-            if (this.outstandingWebsocketResponses > 0) {
-                this.outstandingWebsocketResponses--;
-            }
-            if (this.outstandingWebsocketResponses === 0) {
-                this.isSaving = false;
-            }
-            return;
-        }
-        if (quizSubmission.submitted) {
-            this.outstandingWebsocketResponses = 0;
-            this.isSaving = false;
-            this.unsavedChanges = false;
-            this.isSubmitting = false;
-            this.submission = quizSubmission;
-            this.updateSubmissionTime();
-            this.applySubmission();
-        } else if (this.outstandingWebsocketResponses === 0) {
-            this.isSaving = false;
-            this.unsavedChanges = false;
-            this.submission = quizSubmission;
-            this.updateSubmissionTime();
-            this.applySubmission();
-        } else {
-            this.outstandingWebsocketResponses--;
-            if (this.outstandingWebsocketResponses === 0) {
-                this.isSaving = false;
-                this.unsavedChanges = false;
-                if (quizSubmission) {
-                    this.submission.submissionDate = quizSubmission.submissionDate;
-                    this.updateSubmissionTime();
-                }
-            }
+            // if (this.outstandingWebsocketResponses > 0) {
+            //     this.outstandingWebsocketResponses--;
+            // }
+            // if (this.outstandingWebsocketResponses === 0) {
+            //     this.isSaving = false;
+            // }
         }
+        // if (quizSubmission.submitted) {
+        //     // this.outstandingWebsocketResponses = 0;
+        //     this.isSaving = false;
+        //     this.unsavedChanges = false;
+        //     this.isSubmitting = false;
+        //     this.submission = quizSubmission;
+        //     this.updateSubmissionTime();
+        //     this.applySubmission();
+        // } else if (this.outstandingWebsocketResponses === 0) {
+        //     this.isSaving = false;
+        //     this.unsavedChanges = false;
+        //     this.submission = quizSubmission;
+        //     this.updateSubmissionTime();
+        //     this.applySubmission();
+        // } else {
+        //     this.outstandingWebsocketResponses--;
+        //     if (this.outstandingWebsocketResponses === 0) {
+        //         this.isSaving = false;
+        //         this.unsavedChanges = false;
+        //         if (quizSubmission) {
+        //             this.submission.submissionDate = quizSubmission.submissionDate;
+        //             this.updateSubmissionTime();
+        //         }
+        //     }
+        // }
```

Comment:
That's quite a lot of code commented out. Can it be deleted or is left there on purpose?

Reply:
I wanted to keep it in case we have to revert some of the changes. We will clean it up after all quiz problems are resolved